.storybook/config.js

```diff
@@ -57,6 +58,66 @@ export const wInfo = text =>
   withInfo({ inline: true, source: true, styles: wInfoStyle, text: text });
 
 addDecorator(withKnobs);
+
+const GlobalStyle = createGlobalStyle`
+  html, body{
+    font-weight: ${defaultTheme.typography.fontWeight};
+    font-family: ${defaultTheme.typography.fontFamily};
+    font-size: ${defaultTheme.typography.fontSize};
+    color: ${defaultTheme.colors.primary}
+    line-height: ${defaultTheme.typography.lineHeight};
+  }
+
+  pre, code{
+    white-space: pre;
+    white-space: pre-wrap;
+    white-space: -moz-pre-wrap;
+    white-space: -o-pre-wrap;
+  }
+
+  pre{
+    font-family: ${defaultTheme.typography.pre.fontFamily};
+    font-size: ${defaultTheme.typography.pre.fontSize};
+    line-height: ${defaultTheme.typography.pre.lineHeight};
+  }
+
+  code{
+    font-family: ${defaultTheme.typography.code.fontFamily};
+    font-size: ${defaultTheme.typography.code.fontSize};
+    line-height: ${defaultTheme.typography.code.lineHeight};
+  }
+
+  #content code{
+    display: block;
+    padding: 1.5em 2em;
+    border: 1px solid 
+  }
+
+  .tippy-popper[x-placement^=bottom] .tippy-arrow{
```

Comment:
I'm not sure we want these tippy styles here. How does the popover component behave outside of storybook? Are the correct styles applied?